theme-dark.scss

```
@@ -0,0 +1,8 @@
+/*-- scss:defaults --*/
+$body-bg: #181818;
+$body-color: white;
```

```suggestion
$body-color: #d0d0d0;
```
Should we change the text color to a very light gray? Plain white looks a bit rough, I think.